api/WebSocket.json

```diff
@@ -0,0 +1,732 @@
+{
+  "api": {
+    "WebSocket": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/WebSocket",
+        "support": {
+          "webview_android": {
```

Comment:
Ideally `webview_android` would be at the bottom so this list is sorted alphabetically, but that's probably not really a blocker.